tests/vmi_multiqueue_test.go

```diff
@@ -123,5 +124,33 @@ var _ = Describe("[Serial]MultiQueue", func() {
 			}
 		})
 
+		It("should be able to create a multi-queue VMI when requesting a single vCPU", func() {
+			vmi := libvmi.NewCirros(
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()))
```

Comment:
Every VMI gets a network if none is defined.

Reply:
Doesn't that depend on the `autoAttachPodInterface` attribute value (actually, to it's default value) ? https://github.com/kubevirt/kubevirt/blob/ea872b751a866f144a3ee8ed36e86070addf9149/pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator.go#L152

I'd rather be specific and say that I need the VM to have an interface and 1 network, but I can just leave this up to the  current default.